src/plugins/integrations/pythonRequirementsTxt.ts

```diff
@@ -2,7 +2,7 @@ import { readFileSync } from 'fs';
 import { SiloDiscoveryConfig } from '../types';
 
 export const pythonRequirementsTxt: SiloDiscoveryConfig = {
-  supportedFiles: ['requirements.txt'],
+  supportedFiles: ['**.txt'],
```

Comment:
i think this will likely too much -- we should find a way for the user to specify paths, not just scan any text file?

Reply:
I was thinking instead maybe any file that has a 'requirements' in it?

Reply:
Is supportedFiles already an arg/can already take a glob?

Reply:
If so i feel like we don't need to change anything, just need to tell them hey specify your own glob to be `**.txt` ?

Reply:
They want to be able to specify the specific additional file(s) to be imported - he mentioned being able to specify the additional file paths, but from my perspective specifying the list of files or a glob would be work, he's definitely capable of doing it either way. But I agree that expanding to include any .txt file will likely create too much noise.

Reply:
I see. In that case @jwahjudi I recommend allowing the super to specify additional file paths or globs (comma-separated?) so that we can concat with the default `supportedFiles`. Basically something like:

```
yarn tr-discover-silos --scanPath=./myJavascriptProject --fileGlobs=requirements*.txt,someOtherName.txt --auth=asd123 --dataSiloId=abcdefg
```

Reply:
I dont think we have a support for that yet, I could add that if necessary?

Reply:
ya let's add support for that

Reply:
Hmm I can't seem to pass in the '*' mark as a parameter, I was able to pass in file names perfectly fine, but when it comes to stuff like '**.txt' I keep on getting this error.
`zsh: no matches found: --fileGlobs=**.txt`

Reply:
Actually, I just need to enclose it with an apostrophe in order to set it as an argument like
`--fileGlobs='**.txt'`